Share with Gifski/ShareViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ShareViewController.swift
+//  Share with Gifski
+//
+//  Created by Koray Koska on 12.09.19.
+//  Copyright © 2019 Sindre Sorhus. All rights reserved.
+//
+
+import Cocoa
+import WebKit
+
+class ShareViewController: NSViewController {
+
+    override var nibName: NSNib.Name? {
+        return NSNib.Name("ShareViewController")
+    }
+
+    override func loadView() {
+		super.loadView()
+
+		guard let item = (self.extensionContext?.inputItems[0] as? NSExtensionItem)?.attachments?.first else {
+			errorOpenMainApp()
+			return
+		}
+
+		var typeIdentifier: String
+		if item.hasItemConformingToTypeIdentifier("public.mpeg-4") {
+			typeIdentifier = "public.mpeg-4"
+		} else if item.hasItemConformingToTypeIdentifier("com.apple.m4v-video") {
+			typeIdentifier = "com.apple.m4v-video"
+		} else if item.hasItemConformingToTypeIdentifier("com.apple.quicktime-movie") {
+			typeIdentifier = "com.apple.quicktime-movie"
+		} else {
+			errorOpenMainApp()
```

Comment:
I think this should pass a more useful error to the main app. Instead of making the `error` query parameter a bool, just make it the error message.